### ISSUE TITLE
ENH: Consider whether using a bot has any benefit

### ISSUE THREAD

**Issue by nathanshammah** [qutip/qutip#1089]
I was wondering if configuring a bot might be beneficial for some tasks. For example, checking that the code is well formatted with `black`. Examples are given by Sympy and Cirq (although there they use it for CLAs). 

https://github.com/topics/bot

This could also be a fun project for a student (or coder in general) willing to contribute to QuTiP on the software side instead of the physics side. 

Maybe though, any interesting feature of a bot can be put in place by a given specific API.

**Comment by TanviPooranmal** [comment]
Yes, configuring a bot is beneficial for several tasks. However, the scope would be broad then. Narrowing down seems reasonable. What do you think about opening a new issue, mentioning several examples that can be worked on?

**Comment by veronikakurth** [comment]
The agreement in the admin team is that it's nice to have bots from which we can profit (e.g. automatic code reviews similar to codeclimate tool we used to have) but I'd suggest to close this issue since it's formulated very generally although being nice-to-have, it is impossible for now to define a timeline for its completion. We welcome contributions with specific proposals, though!
